### PR TITLE
Chase Pay Transaction changes in sample code

### PR DESCRIPTION
**Pull request by nagarikantipavan**

### CODE REVIEW

File: src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        creditCard.setTokenRequestorEci("07");
+        creditCard.setCardCode("999");
+        paymentType.setCreditCard(creditCard);
+
+        // Create the payment transaction request
+        TransactionRequestType txnRequest = new TransactionRequestType();
+        txnRequest.setTransactionType(TransactionTypeEnum.AUTH_CAPTURE_TRANSACTION.value());
+        txnRequest.setPayment(paymentType);
+        txnRequest.setAmount(new BigDecimal(amount).setScale(2, RoundingMode.CEILING));
+        //txnRequest.s
```
Comment (gnongsie) [inline src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java]: Remove this line

File: src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        // Make the API Request
+        CreateTransactionRequest apiRequest = new CreateTransactionRequest();
+        apiRequest.setTransactionRequest(txnRequest);
+        //apiRequest.setRefId("123456");
```
Comment (gnongsie) [inline src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java]: Remove this line

File: src/main/java/net/authorize/sample/SampleCode.java [hunk 3]
```diff
@@ -343,6 +344,9 @@ private static void RunMethod(String methodName)
                 break;
             case "GetAnAcceptPaymentPage":
             	GetAnAcceptPaymentPage.run(apiLoginId, transactionKey, amount);
+                break;
+			case "CreateChasePayTransaction":
+            	CreateChasePayTransaction.run(apiLoginId, transactionKey, amount);
```
Comment (gnongsie) [inline src/main/java/net/authorize/sample/SampleCode.java]: Fix indentation in these lines

File: src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java (hunk 1 above)
Comment (nagarikantipavan) [reply src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java]: I have removed the line that is mentioned and I have pushed as well.

File: src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java (hunk 2 above)
Comment (nagarikantipavan) [reply src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java]: I have removed the line that is mentioned and I have pushed as well.

File: src/main/java/net/authorize/sample/SampleCode.java (hunk 3 above)
Comment (nagarikantipavan) [reply src/main/java/net/authorize/sample/SampleCode.java]: I have made the changes and I have pushed.

**Comment by gnongsie** [comment]
Ready to merge

**Pull request merged**